signac/contrib/project.py

```diff
@@ -789,9 +785,9 @@ def _contains_job_id(self, job_id):
             True if the job id is initialized for this project.
 
         """
-        # We can rely on the project workspace to be well-formed, so just use
-        # str.join with os.sep instead of os.path.join for speed.
-        return os.path.exists(os.sep.join((self.workspace(), job_id)))
+        # We can rely on the project workspace to be well-formed, so use
+        # os.sep.join instead of os.path.join for performance.
```

Comment:
The changes in this comment aren't aligned with the comment in `Job.path`. Either is fine but I slightly prefer the other for clarifying "`str.join` with `os.sep`":

```suggestion
        # We can rely on the project workspace to be well-formed, so just
        # use str.join with os.sep instead of os.path.join for speed.
```

Reply:
Sure.